src/cpu/s390x/gemmu16.cpp

```diff
@@ -0,0 +1,279 @@
+#include <cstdint>
```

Comment:
Could you please put an Apache license at the beginning of the files? Here is an example: https://github.com/oneapi-src/oneDNN/blob/master/src/cpu/ppc64/ppc64_gemm_driver.hpp